recipes/libcurl/all/conanfile.py

```diff
@@ -244,6 +244,11 @@ def _patch_mingw_files(self):
                                   '-lz ',
                                   '-lzlib ')
 
+        # patch for openssl extras in mingw
+        tools.replace_in_file("configure",
```

Comment:
Should this be conditioned to `with_openssl=True`?

Reply:
no difference, because -lcrypto in configure is wrapped by SSL option. Which means, we will update the configure file, but it will only run if with_openssl is True. Anyway, I liked your suggestion, I think will be better putting a condition to make it more explicit